crates/aptos/src/common/init.rs

```diff
@@ -202,7 +202,7 @@ impl CliCommand<()> for InitTool {
                 let input = input.trim();
                 if input.is_empty() {
                     if let Some(key) = profile_config.private_key {
-                        eprintln!("No key given, keeping existing key...");
+                        eprintln!("Private key given, keeping existing key...");
```

Comment:
Thanks for this contribution!

Maybe a more appropriate message is:
```
No private key given, keeping existing private key in config...
```

Reply:
@gregnazario thx, see my suggestion below